docs/presigned-put-upload-via-browser.md

```diff
@@ -1,25 +1,34 @@
-# Upload files from browser using pre-signed URLs [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+# Upload Files from a Browser Using Pre-signed URLs [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 
-Using presigned URLs, you can allow a browser to upload a file
-directly to S3 without exposing your S3 credentials to the user. The following
-is an annotated example of this using [minio-js](https://github.com/minio/minio-js).
+Using pre-signed URLs, a browser can upload files directly to an S3-compatible cloud storage server (S3) without exposing the S3 credentials to the user. 
 
-### Server code
+This guide describes how to use the [`presignedPutObject`](https://docs.minio.io/docs/javascript-client-api-reference#presignedPutObject) API from the [Minio JavaScript Library](https://github.com/minio/minio-js) to generate a pre-signed URL. This is demonstrated through a JavaScript example in which an Express Node.js server exposes an endpoint to generate a pre-signed URL and a client-side web application uploads a file to Minio Server using that URL.
+
```

Comment:
Since its a client-side webapp, i feel ambivalent about saying a browser uploads via pre-signed URL

Reply:
@deekoder I have made this change in the branch. Please see https://github.com/jamesneimanconsulting/cookbook/blob/patch-7/docs/presigned-put-upload-via-browser.md.